src/items/item.js

```diff
@@ -112,7 +128,12 @@ export class Item extends ClickableObject {
 
     drop() {
         console.log("Drop", this.name);
-        this.destroy();
```

Comment:
I'm considering replacing the "drop" command with "sell". What do you think about that? Also I can make it so when the item count is > 1 it can be sell 1 or sell all. Or do you think this would overinflate the user's gold (which is already pretty inflated)

Reply:
Only giving gold on monster death would help solve the inflation. Sell should probably only be when the shop is open

Reply:
Not sure what happened to the reponse I did. But right now the shop doesn't show inventory when it's open, if we want to sell only when the shop is open we would need to redesign the shop. If you have a good idea for a layout I can implement it. Only reason I thought of replacing "drop" with "sell" is because I don't see any reason why the user would want to drop something, unless we never want the user to sell anything

Reply:
aight we can change to sell, but if we do that then might want a sell X item for stackable items